### PR TITLE
[stable/clamav] Let the user choose the api version of their deployment.

### PR DESCRIPTION
**Pull request by distortedsignal**
#### Is this a new chart
No

#### What this PR does / why we need it:
When a user attempts to install clamav to their kubernetes cluster, if the kubernetes version is 1.9+, the install will fail. This happens because [the Deployments resource moved to `apps/v1` in kubernetes 1.9](https://github.com/kubernetes/kubernetes/issues/55894#issuecomment-361062485).

#### Checklist
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/mychartname]`)

#### DCO
```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.

Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```
Signed-off-by: Thomas Kelley <distortedsignal@gmail.com>

### CODE REVIEW

File: stable/clamav/README.md [hunk 1]
```diff
@@ -32,6 +32,34 @@ The command deploys ClamAV on the Kubernetes cluster in the default configuratio
 
 > **Tip**: List all releases using `helm list`
 
+In order to deploy this chart under Kubernetes 1.9+, the `kubeMeta.deploymentApiVersion` MUST be set to "apps/v1":
+
+```bash
+$ helm install --set "kubeMeta.deploymentApiVersion=apps/v1" --name my-release stable/clamav
+```
+
+In order to deploy this chart behind a firewall that requires an HTTP Proxy, create a template file as below and install passing the `-f` flag:
```
Comment (zakkg3) [inline stable/clamav/README.md]: why this if you also included in the values files.

File: stable/clamav/README.md [hunk 2]
```diff
@@ -32,6 +32,34 @@ The command deploys ClamAV on the Kubernetes cluster in the default configuratio
...
+  PidFile /run/clamav/freshclam.pid
+  DatabaseOwner root
+  DatabaseMirror database.clamav.net
+  ScriptedUpdates yes
+  NotifyClamd /etc/clamav/clamd.conf
+  SafeBrowsing yes
+  Bytecode yes
+EOF
+
+$ helm install -f ./template.yaml -name my-release stable/clamav
```
Comment (zakkg3) [inline stable/clamav/README.md]: plase remove your changes from loines 36 the readme.
If we include the config changes in values then `helm instal stable/clamav` will be enough. keep it simple.

File: stable/clamav/Chart.yaml [hunk 3]
```diff
@@ -2,9 +2,11 @@ apiVersion: v1
...
 description: An Open-Source antivirus engine for detecting trojans, viruses, malware & other malicious threats.
 name: clamav
-version: 1.0.4
+version: 1.0.5
 home: https://www.clamav.net
 icon: https://www.clamav.net/assets/clamav-trademark.png
 maintainers:
 - name: zakkg3
   email: zakkg3@gmail.com
+- name: distortedsignal
```
Comment (zakkg3) [inline stable/clamav/Chart.yaml]: why?

**Comment by distortedsignal** [comment]
Hey @zakkg3 - I’m not sure what comments you’re referring to here - could you be more specific?

**Pull request merged**